api/envoy/extensions/filters/http/grpc_field_extraction/v3/config.proto

```diff
@@ -97,7 +97,8 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // }
 
 message GrpcFieldExtractionConfig {
-  // descriptor_set is in local data source. It could be either in a local file or embedded in the
+  // The proto descriptor set for the gRPC services.
```

Comment:
In what form?  As a text proto?  Binary form?

What bytes would I be expected to put in the file if I were populating it?

Reply:
Updated. PTAL.